apps/ui/src/components/App/Nav.vue

```diff
@@ -42,15 +43,17 @@ const space = computed(() =>
     : null
 );
 
-const isController = computedAsync(async () => {
-  if (!networkId.value || !space.value) return false;
-
-  const { account } = web3.value;
+const controller = computedAsync(async () => {
+  if (!networkId.value || !space.value) return null;
 
   const network = getNetwork(networkId.value);
-  const controller = await network.helpers.getSpaceController(space.value);
+  return await network.helpers.getSpaceController(space.value);
```

Comment:
```suggestion
  return network.helpers.getSpaceController(space.value);
```

`await` there does nothing.